projects/Apps/yarn.lock

```diff
@@ -1220,6 +1220,111 @@
     "@types/yargs" "^15.0.0"
     chalk "^3.0.0"
 
+"@react-native-community/netinfo@^5.6.2":
```

Comment:
Just checking there is no uneccessary additions here? Or something that shouldnt have changed?

Reply:
Had a look through and I think they're ok - I think all the changes are from the modificatoins to the `common` package.json